WordPress/src/main/java/org/wordpress/android/ui/uploads/VideoOptimizerProvider.kt

```diff
@@ -0,0 +1,8 @@
+package org.wordpress.android.ui.uploads
+
+// TODO: move here the VideoOptimizationListener interface and ProgressEvent. Leaving them in their original
+// position for now to not change the files as they are
```

Comment:
👍 agreed, it might be good to have them together

Reply:
Moved interfaces and data classe used by them to VideoOptimizerContracts.